bucket/transgui.json

```
@@ -0,0 +1,31 @@
+{
+    "homepage": "https://github.com/transmission-remote-gui/transgui",
+    "description": "A feature rich cross platform Transmission BitTorrent client. Faster and has more functionality than the built-in web GUI.",
+    "version": "5.16.0",
+    "license": "GPL-2.0-or-later",
+    "url": "https://github.com/transmission-remote-gui/transgui/releases/download/v5.16.0/transgui-5.16-setup.exe",
+    "hash": "630691d1cdc62145b78b4e1578adf0826026c3ba3a98ef1cacf23a919aeacd73",
+    "innosetup": true,
+    "bin": "transgui.exe",
+    "pre_install": "if(!(Test-Path \"$dir\\transgui.ini\")) {New-Item (\"$dir\\transgui.ini\") -Type File | Out-Null}",
```

- Array at new-item is nonsense.
- Use cross persist dir and dir check
```suggestion
    "pre_install": "if(!(Test-Path \"$persist_dir\\transgui.ini\")) { New-Item \"$dir\\transgui.ini\" -Type File | Out-Null }",
```